docs/pages/docs/components/toggle.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: "Toggle"
+description: "Displays a boolean value."
+---
+
+import Playground from "@components/Playground";
+
+# Toggle
+
+Displays a boolean value.
+
+## Usage
+
+First of all, you need to import the `Toggle` component from the `@tonightpass/kitchen` package.
+
+```js
+import { Toggle } from "@tonightpass/kitchen"
+```
+
+## Default
+
+<Playground
+  code={`
+  () => {
+  const [checked, toggle] = useCheckbox();
+  const [checked1, toggle1] = useCheckbox(true);
```

Comment:
this is second checked so replace `[checked1, toggle1]` by `[checked2, toggle2]`